api/db/changelog/db.changelog-10.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="calbach" id="changelog-10">
+    <addColumn tableName="workspace">
+      <column name="version" type="smallint">
+        <constraints nullable="false"/>
```

Comment:
@danqrodney I added this at the last second. I'm not clear on what will happen when this migration happens and how it will interact with my model field which is defined as an `int`. Namely, I don't know if liquibase will insert a default value here for us, or if not, whether the JPA models will read us out correctly.

I can test out these scenarios but wanted to check if there's a convention.

Reply:
I think you''l need defaultValue="1" on the column tags. You should be able to run without this migration, add a workspace, and then apply it to verify the behavior.

Alternatively we can have everybody reset their database when this PR goes in.